dts/bindings/sensor/ti,tmp1075.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2024 Arrow Electronics.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  TMP1075 Digital Temperature Sensor.  See more info at
+  https://www.ti.com/product/TMP1075
+
+compatible: "ti,tmp1075"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  conversion-rate:
+    description: Conversion rate in ms
```

Comment:
if 275 is 27.5ms, then to be precise this is Conversion rate in 100us (or 0.1ms)

Reply:
these are numbers from datasheet, ill leave it as it is - added more comments